components/dashboard/src/Menu.tsx

```diff
@@ -230,10 +233,10 @@ export default function Menu() {
                     </div>
                 </div>
             </div>
-            {showTeamsUI && <div className="flex">
+            {!isMinimalUI && showTeamsUI && <div className="flex">
                 {leftMenu.map(entry => <TabMenuItem name={entry.title} selected={isSelected(entry, location)} link={entry.link}/>)}
             </div>}
         </header>
```

Comment:
**question**: What do you think of restoring the bottom border to ad some visual balance when creating a new team or project? 💭 

| BEFORE | AFTER | 
|-|-|
|  <img width="1440" alt="Screenshot 2021-06-25 at 2 05 11 PM" src="https://user-images.githubusercontent.com/120486/123415782-692d2180-d5be-11eb-8a1c-5471ec471c15.png"> | <img width="1440" alt="Screenshot 2021-06-25 at 2 05 14 PM" src="https://user-images.githubusercontent.com/120486/123415787-6af6e500-d5be-11eb-9e77-d5a9cc4d8030.png"> |

Reply:
Sure! Happy to try that.

Reply:
Now looks like this:

<table><tr><td>
<img width="1440" alt="Screenshot 2021-06-25 at 16 18 12" src="https://user-images.githubusercontent.com/599268/123438469-03966080-d5d1-11eb-8032-60dbf6a064a9.png">
</td><td>
<img width="1440" alt="Screenshot 2021-06-25 at 16 18 20" src="https://user-images.githubusercontent.com/599268/123438487-05f8ba80-d5d1-11eb-8d20-8730e0eafa62.png">
</td></tr></table>